### DISCUSSION TITLE
Make chat & suggestion window sizes and positions stick

### DISCUSSION THREAD

**Discussion by czottmann** [intitni/CopilotForXcode#421] [Ideas]
I'd love to be able to use the chat and suggestions window without having to resize them every time I re-open them.

I often have more than one window open (editor, log, prefs, etc.), and re-opening the chat will make it latch to the current window, whereas I'd like to always have it pop up in the same spot, in the same size. Actually, I don't want it to stick to one particular editor, that's just the way I work. 😉

Thanks, and keep up the good work!

**Comment by intitni** [comment]
I think it's a bug that a detached chat window gets resized and moved after it's being displayed from hidden. Please open an issue for it.

**Reply by intitni** [reply]
Fixed in #422 

Looks like it was implemented as a feature.

**Reply by czottmann** [reply]
Oh, nice, thank you!

**Reply by intitni** [reply]
Released in 0.30.1

**Reply by czottmann** [reply]
Thanks, appreciate the speedy change! 🖖🏼

**Comment by intitni** [comment]
I saw that you have also mentioned the suggestion window. But I don't think they can be resized?

**Comment by czottmann** [comment]
Thanks for the pointers above! Correct, the suggestion window can't be resized but I'd like it to be. :)
